mlflow/catboost/__init__.py

```diff
@@ -335,6 +335,12 @@ class _CatboostModelWrapper:
     def __init__(self, cb_model):
         self.cb_model = cb_model
 
+    def get_raw_model(self):
```

Comment:
We might want to add an abstract class. Not a blocker.

Reply:
Yes we really need one! I can file a follow-up PR for this. While `get_raw_model` not always exists, for example `promptlab` it has `model_route` but technically it is not a model.